Mail/SecurityOtpMail.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+namespace WebReinvent\VaahCms\Mail;
+
+use Illuminate\Bus\Queueable;
+use Illuminate\Contracts\Queue\ShouldQueue;
+use Illuminate\Mail\Mailable;
+use Illuminate\Queue\SerializesModels;
+
+class SecurityOtpMail extends Mailable
+{
+    use Queueable, SerializesModels;
+
+    public $content;
+
+    /**
+     * Create a new message instance.
+     *
+     * @return void
+     */
+    public function __construct($content)
+    {
+        $this->content = $content;
+    }
+
+    /**
+     * Build the message.
+     *
+     * @return $this
+     */
+    public function build()
+    {
+        return $this->subject(config('app.name').': Two Step Authentication')
+            ->view('vaahcms::mails.security-otp');
```

Comment:
@we-vikram-wri231 do we really need to create another view file `security-opt`? we could have used `generic`, explain

Reply:
@themodernpk Sir, we used some template to show OTP Code as we discussed while development.